fiftyone/core/collections.py

```diff
@@ -1887,7 +1888,10 @@ def _get_visualization_keys(self, **kwargs):
     def _get_brain_runs_with_type(self, run_type, **kwargs):
         brain_keys = []
         for brain_key in self.list_brain_runs():
-            brain_info = self.get_brain_info(brain_key)
+            try:
+                brain_info = self.get_brain_info(brain_key)
+            except:
+                continue
```

Comment:
Why hide these exceptions? A comment might be helpful.

Reply:
Yeah good point. I didn't want to raise the error because I don't want one unusable brain result to affect the ability to affect other ones. But, it wouldn't hurt to warn the user that something was not usable, even if they continually see it until they finally delete that brain result from their dataset (or upgrade, etc)